AUTHENTICATION_QUICKSTART.md

```diff
@@ -0,0 +1,236 @@
+# Authentication Quick Start Guide
+
+This guide will help you quickly set up and test Webex authentication for the BYOVA Gateway monitoring dashboard.
+
+## Prerequisites
+
+- Python 3.8+ with virtual environment activated
+- Access to [Webex Developer Portal](https://developer.webex.com/)
+- A Webex account
+
+## Step 1: Create a Webex Integration
+
+1. Go to https://developer.webex.com/
+2. Log in with your Webex account
+3. Click **"Create an Integration"** (under "My Apps")
+4. Fill in the form:
+   - **Integration Name**: BYOVA Gateway Monitor (Dev)
+   - **Icon**: Upload an icon or use default
+   - **Description**: Development environment for BYOVA Gateway monitoring
+   - **Redirect URI**: `http://localhost:8080/oauth`
+   - **Scopes**: Check `openid`, `email`, and `profile`
+5. Click **"Add Integration"**
+6. **Save your Client ID and Client Secret** - you'll need these next
+
+## Step 2: Find Your Organization ID
+
+### Method 1: Using Access Token Parsing
+
+1. Get your Webex access token from the Developer Portal (click your profile icon)
+2. The access token format is: `{access_token}_{ci_cluster}_{org_id}`
+3. Split the token by underscores and take the third part
+4. Copy the organization ID (the third segment after splitting)
+
+Example in a browser console or terminal:
+```javascript
+// JavaScript
+let accessToken = "your_access_token_here";
+let parts = accessToken.split('_');
+let orgId = parts[2];
+console.log("Organization ID:", orgId);
+```
+
+```bash
+# Bash
+echo "your_access_token_here" | awk -F'_' '{print $3}'
+```
+
+### Method 2: Using Webex API
+
+1. Go to https://developer.webex.com/docs/api/v1/people/get-my-own-details
+2. Click "Run" in the API Reference
+3. Look for `orgId` in the response
+4. Copy the organization ID
+
+## Step 3: Set Environment Variables
+
+### For macOS/Linux (bash/zsh):
+
+```bash
+# Generate a random secret key
+export FLASK_SECRET_KEY="$(openssl rand -hex 32)"
+
+# Set Webex OAuth credentials
+export WEBEX_CLIENT_ID="your-client-id-from-step-1"
+export WEBEX_CLIENT_SECRET="your-client-secret-from-step-1"
+export WEBEX_REDIRECT_URI="http://localhost:8080/oauth"
+
+# Set your organization ID
+export AUTHORIZED_WEBEX_ORG_IDS="your-org-id-from-step-2"
```

Comment:
Could you add more details on setting this field. It is the allow list of the site and the documentation on it is a bit lacking. I see that it supports multiple IDs, what format should those be? Comma, spaced, etc.